bin/src/command_line.dart

```diff
@@ -1,17 +1,47 @@
 library cmdpart;
 
 import 'dart:async';
+import 'dart:io' show FileSystemEntity;
 import "dart:math" show max;
 
 import "package:args/args.dart";
+import 'package:path/path.dart' as p;
+import 'package:logging/logging.dart';
 
 export "package:args/args.dart";
 
+final Logger _log = new Logger("dart_coveralls");
+
 abstract class CommandLinePart {
   final ArgParser parser;
 
   CommandLinePart(this.parser);
 
+  String handlePackagesArg(ArgResults res) {
+    String packagesPath = res["packages"];
+    if (p.isRelative(packagesPath)) {
+      packagesPath = p.absolute(packagesPath);
+    }
+
+    if (!FileSystemEntity.isFileSync(packagesPath)) {
+      print("Packages file does not exist");
+      return null;
+    }
+
+    _log.info(() => "Packages file is ${packagesPath}");
+
+    return packagesPath;
+  }
+
+  static ArgParser addCommonOptions(ArgParser parser) {
+    return parser
+      ..addFlag("help", help: "Displays this help", negatable: false)
+      ..addOption("packages",
```

Comment:
I'm unsure about the effects of simply removing the option `--package-root` in a travis environment, for example. Some official Dart packages (e.g. `stagehand` used by the Dart intellij-plugin` invoke `dart_coveralls` via travis-ci.org - does this change introduce a difference in exception handling, causing travis tests to fail, and, as a logical consequence, the internet to break? If yes, have fun explaining to your colleagues at work ;-)

Your implementation would support both ways to reference packages, see comment further down.